lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -403,14 +403,22 @@ public void setUseCompositionFrameRate(boolean useCompositionFrameRate) {
    * instead of using merge paths.
    */
   public void enableMergePathsForKitKatAndAbove(boolean enable) {
-    lottieDrawable.enableMergePathsForKitKatAndAbove(enable);
+    lottieDrawable.enableFeatureFlag(LottieFeatureFlags.FeatureFlag.MergePathsApi19, enable);
   }
 
   /**
    * Returns whether merge paths are enabled for KitKat and above.
    */
   public boolean isMergePathsEnabledForKitKatAndAbove() {
-    return lottieDrawable.isMergePathsEnabledForKitKatAndAbove();
+    return lottieDrawable.isFeatureFlagEnabled(LottieFeatureFlags.FeatureFlag.MergePathsApi19);
+  }
+
+  public void enableFeatureFlag(LottieFeatureFlags.FeatureFlag flag, boolean enable) {
```

Comment:
Could you add docs here since this is a new public API?

I would make it extra explicit that people validate that opted in features look acceptable across all supported API levels.